omnibus/package-scripts/agent-rpm/postrm

```diff
@@ -1,26 +0,0 @@
-#!/bin/sh
-#
-# Perform necessary datadog-agent removal steps after package is uninstalled.
-#
-# .rpm: STEP 5 of 6
-
-CONFIG_DIR=/etc/datadog-agent
-
-# Remove the symlink to the binary.
-rm -f "/usr/bin/datadog-agent"
```

Comment:
Where is this handled from now?

Reply:
nice catch, missed moving it to the remove flow

Reply:
fixed now